docs/sources/writing-guide/contribute-release-notes/index.md

```diff
@@ -12,6 +12,127 @@ keywords:
+
+### Examples of what *not* to include in What’s New
+
+These are important improvements, but are better placed in the changelog than What’s new:
+
+- [A docs update](https://github.com/grafana/grafana/pull/60352)
+  - This update doesn’t require customers to change their behavior - they’ll simply see better instructions the next time they use the docs.
+- [A bug fix related to migrations](https://github.com/grafana/grafana/pull/59438)
+  - This is a bug fix that doesn’t require customer action.
+- [A usability improvement to an existing transformation](https://github.com/grafana/grafana/pull/59074)
+  - Nice fix, but very detailed. Should be in the changelog but not What’s New.
+- [Change regex to accommodate a new branching strategy in Enterprise](https://github.com/grafana/grafana/pull/59429)
+  - This change is invisible to customers.
+
+## What's New doc development process
+
+Developing directly in Markdown reduces errors and removes a manual (toilsome and error-prone step), by creating, editing, discussing, and publishing in the same format from which we will publish in GitHub, using the Grafana repo. 
+
+We also recommend that you think critically about whether to include an image in the What’s New document. We will not maintain these images, as the document refelct a "point in time". If you need to add an image, refer to [Image, diagram, and screenshot guidelines]({{</* relref "../image-guidelines" */>}}).
+
+### When the prior release is complete
+
+Complete the following steps after the prior release goes out.
+
+1. The Docs Squad identifies the technical writer responsible for overseeing the What’s New content development process for the upcoming release.
+1. The identified technical writer cuts a branch and creeates a PR with an empty What's New doc that will be populated with the What’s New content for the next release. 
+1. The technical writer then pins a link to [version-number-release-gtm](https://raintank-corp.slack.com/archives/C02L8GQJGBW) Slack channel.
+
+   > **Note:** A Product Manager (PM) is responsible for update the version number of that channel after a release goes out. 
+   
+1. The technical writer posts in the Slack channel that the What's New PR is ready for populating. 
+1. The PM and Product Marketing Manager (PMM) organizations identify which PM or PMM is responsible for vetting content added to the What's New draft.
+1. The PM or PMM on point communicates with eng squads without a PM resource about who’s on point for What’s New PR reviews from within the PM and PMM orgs.
+
+### Throughout a release
+
+A Grafana release cycle is typically 6-8 weeks, and during that time, developers are empowered to add to the What's New document incrementally. To see a list of release dates and assigned PMs and PMMs, refer to [Grafana Releases and Release Dates](https://grafana-intranet--simpplr.visualforce.com/apex/simpplr__app?u=/site/a145f000001dCXBAA2/page/a125f000001AXrwAAG).
```

Comment:
@mjseaman, does that release cadence sound about right?

Reply:
Yes 👍